redis/redis-client.js

```diff
@@ -8,5 +8,53 @@ client.on('error', function (err) {
   console.log('Error ' + err);
 });
 
-// write stub for retrieving a productObj from Redis
+// ============ HELPER FOR RETRIEVING FROM REDIS  ================
 
+module.exports.getProduct = (productId) => {
+  // look for productObject in Redis
+  // if it's in Redis, return it
+  // if it isn't, return undefined;
+};
+
+// ====== HELPER TO PARSE INV RES INTO PRODUCT JS OBJECTS =========
+
+module.exports.parseProduct = (req) => {
+  var prodObj = {};
+  var pd = req.body.prodDetailsQA[0];
```

Comment:
What does `pd` and `pr` mean? These variables names are very short especially in comparison to `prodObj` is long.